src-java/nbworker-topology/nbworker-storm-topology/src/main/java/org/openkilda/wfm/topology/nbworker/bolts/FlowOperationsBolt.java

```diff
@@ -294,15 +288,11 @@ public void emitPeriodicPingUpdate(String flowId, boolean enabled) {
 
     @Override
     public void sendRerouteRequest(Collection<FlowPath> paths, Set<IslEndpoint> affectedIslEndpoints, String reason) {
-        boolean flowsRerouteViaFlowHs = featureTogglesRepository.find()
-                .map(FeatureToggles::getFlowsRerouteViaFlowHs)
```

Comment:
Can we remove `flowsRerouteViaFlowHs` feature field too?

Reply:
Yes, we can. I will do it in PR with removing FlowTopology.